kura/org.eclipse.kura.core.cloud/OSGI-INF/metatype/org.eclipse.kura.cloud.CloudService.xml

```diff
@@ -83,7 +83,7 @@
             cardinality="0"
             required="true"
             default="true"
-            description="By disabling default subscriptions the gateway will not be remotely manageable"/>
+            description="Disables the default subscriptions to the gateway management MQTT topics. When disabled, the gateway will not be remotely manageable."/>
             
```

Comment:
Why "Disables the ..."? It is better to use "Enables the..." since the property name is **enable**.default.subscriptions.